Sources/Datadog/RUM/RUMVitals/VitalInfoSampler.swift

```diff
@@ -39,7 +39,7 @@ internal final class VitalInfoSampler {
 
     var refreshRate: VitalInfo {
         let info = refreshRatePublisher.currentValue
-        return info.scaledDown(by: maximumRefreshRate)
+        return info.scaledDown(by: maximumRefreshRate / 60.0)
```

Comment:
How about defining a constant and adding a comment on normalization? I'm afraid that this might become forgotten and unclear.

Reply:
comment added to the line above
> NOTE: `refreshRate` is normalized to 0...60 range, assuming 60 is the industry standard.

Reply:
I agree with using a constant value for this!

Reply:
i can't come up with a local variable name which doesn't make things more confusing 🤔 
any suggestions?

Reply:
’’’swift
struct Constants {
    // We use normalized 0...60 range for refresh rate in Mobile Vitals,
    // assuming 60 is the industry standard.
    static let normalizedRefreshRate = 60
}
’’’
Is +/- what we do in other places.